### DISCUSSION TITLE
ReVanced GMSCore v0.3.1.4.240913 Fails to Connect to Google Servers on Quest 3

### DISCUSSION THREAD

**Discussion by jamesy0ung** [ReVanced/revanced-discussions#1116] [Question] [answered]
I’m using the latest available build of ReVanced GMSCore (v0.3.1.4.240913) on my Quest 3, which I sideloaded from [this link](https://github.com/ReVanced/GmsCore/releases/download/v0.3.1.4.240913/app.revanced.android.gms-240913008-signed.apk). After successfully authenticating, I’m unable to connect to Google servers.

Additionally, I am using YouTube 19.46.42, patched with the ReVanced Patcher. The issue occurs immediately after authentication. I’ve attached an image showing the error I encountered.

![IMG_1687](https://github.com/user-attachments/assets/78d06514-c31b-43b3-aac0-8c1392fd3c07)

**Comment by LisoUseInAIKyrios** [comment]
Try turning off 2 step verification.  Or try using a different YouTube account.

If the issue persists then check for any open or recently closed issues in the upstream project https://github.com/microg/GmsCore

**Reply by jamesy0ung** [reply]
Cheers mate, I'll give that a go.

**Reply by jamesy0ung** [reply, marked as answer]
I tried the latest release (org.microg.gms-240913010-user.apk) and it bricked my quest. Using the huawei version however worked well (com.google.android.gms-240913010-hw.apk). Thanks!

**Reply by LisoUseInAIKyrios** [reply]
And ReVanced YT works correctly with the upstream MicroG?

Try using the ReVanced Huawei build.

**Comment by disconnect5852** [comment]
Same here on Waydroid!
[video about reproducing](https://photos.app.goo.gl/om3Nc8XpvDG7g89f7)

[Closed issue without fix](https://github.com/ReVanced/GmsCore/issues/172)
